charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -285,13 +285,13 @@ data:
       elasticsearch:
         enabled: true
         connection:
-          host: {{ printf "%s-elasticsearch-nginx.%s" .Release.Name .Release.Namespace }}
-          port: 9200
+          node: {{ printf "https://%s-external-es-proxy:9201" .Release.Name }}
+          log: error
```

Comment:
the new additions are for external elasticsearch , retaining the existing ones are correct for this release
host: {{ printf "%s-elasticsearch-nginx.%s" .Release.Name .Release.Namespace }}
port: 9200